src/galaxy/StarSystemGenerator.cpp

```diff
@@ -1369,9 +1372,14 @@ void PopulateStarSystemGenerator::PopulateStage1(SystemBody *sbody, StarSystem::
 			sbody->m_population = fixed(1, 100000) + fixed(starportPopRand.Int32(-1000, 20000), 1000000000);
 			outTotalPop += sbody->m_population;
 		} else if (sbody->GetType() == SystemBody::TYPE_STARPORT_SURFACE) {
-			// give surface spaceports a population between 80000 and 250000
-			sbody->m_population = fixed(1, 10000) + fixed(starportPopRand.Int32(-2000, 15000), 100000000);
-			outTotalPop += sbody->m_population;
+			// No permanent population on gravities larger than about 5 g
+			if (sbody->CalcSurfaceGravity() > 50) {
```

Comment:
instead of '50' this should be saved in a constant variable somewhere. Same for the hard coded surface gravity limits.

Reply:
src/gameconsts.h

Reply:
I generally consider gameconsts appropriate for "facts" about reality like the gravitational constant. This seems like something that'd be better in a galaxy generation related header file, however gameconsts is an acceptable place to put it.

Reply:
Now sits in StarSystemGenerator.cpp as it had a bunch of other variables defined in the top and it's only used there.